### PR TITLE
ISPN-9556 The netty server module depends on the log4j and sun.jdk mo…

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.jboss.org/browse/ISPN-9556

### CODE REVIEW

File: feature-pack/client/src/main/resources/modules/io/netty/slot/module.xml [hunk 1]
```diff
@@ -17,5 +17,7 @@
     <dependencies>
         <module name="javax.api"/>
         <module name="org.javassist" optional="true"/>
+        <module name="org.apache.log4j"/>
```
Comment (danberindei) [inline feature-pack/client/src/main/resources/modules/io/netty/slot/module.xml]: Isn't this module log4j 1.x? I think Netty uses log4j 2.x.

Comment (tristantarrant) [reply feature-pack/client/src/main/resources/modules/io/netty/slot/module.xml]: WildFly comes with neither. It has a JBoss Logging shim that pretends to be log4j [12]

Comment (danberindei) [reply feature-pack/client/src/main/resources/modules/io/netty/slot/module.xml]: Ah ok

**Comment by danberindei** [comment]
Integrated, thanks Tristan

**Pull request merged**